examples/models/llama/source_transformation/quantized_kv_cache.py

```diff
@@ -10,9 +10,36 @@
 import torch
 import torch.nn as nn
 from executorch.examples.models.llama.llama_transformer import KVCache
+
+# This is needed to ensure that custom ops are registered
+from executorch.extension.pybindings import portable_lib  # noqa # usort: skip
+from executorch.extension.llm.custom_ops import custom_ops  # noqa: F401
 from torch.ao.quantization.fx._decomposed import quantized_decomposed_lib  # noqa: F401
 
 
+try:
+    op = torch.ops.quantized_decomposed.quantize_per_token.out
+    assert op is not None
+except:
+    import glob
+
+    import executorch
+
+    # Ideally package is installed in only one location but usage of
+    # PYATHONPATH can result in multiple locations.
+    # ATM this is mainly used in CI for qnn runner. Will need to revisit this
+    executorch_package_path = executorch.__path__[-1]
+    libs = list(
+        glob.glob(
+            f"{executorch_package_path}/**/libquantized_ops_aot_lib.*", recursive=True
+        )
+    )
+    assert len(libs) == 1, f"Expected 1 library but got {len(libs)}"
+    logging.info(f"Loading custom ops library: {libs[0]}")
+    torch.ops.load_library(libs[0])
+    op = torch.ops.quantized_decomposed.quantize_per_token.out
+    assert op is not None
+
```

Comment:
Is this needed? I feel like we can add this to `executorch/kernels/quantized/__init__.py`

Reply:
Oh I didnt realize that this was already there in quantized/__init__.py. I can try to patch that one and see if it works. Althought it does feel quite weird, TBH, to have executorch/kernels/quantized to be exposed as python package. But it does contain AOT component, so seems fair.